src/types/relay.ts

```diff
@@ -0,0 +1,43 @@
+import {SignedOrder} from './types';
+
+export interface RelayerApiTokenTradeInfo {
+    address: string;
+    symbol: string;
+    decimals: number;
+    minAmount: string;
+    maxAmount: string;
+    precision: number;
+}
+
+export interface RelayerApiTokenTradeInfo {
+    tokenA: RelayerApiTokenTradeInfo;
+    tokenB: RelayerApiTokenTradeInfo;
+}
+
+export type OrderState = 'OPEN'|'EXPIRED'|'CLOSED'|'UNFUNDED';
+
+export interface RelayerApiOrderResponse {
+    signedOrder: SignedOrder;
+    state: OrderState;
+    pending: {
+        fillAmount: number;
+        cancelAmount: number;
+    };
+    remainingTakerTokenAmount: number;
+}
+
+export interface RelayerApiFeesRequest {
+    maker: string;
+    taker: string;
+    makerTokenAddress: string;
+    takerTokenAddress: string;
+    makerTokenAmount: string;
+    takerTokenAmount: string;
```

Comment:
So maybe I misunderstand but I feel like this client should also convert the received amounts and convert them to BigNumbers before returning the response to the caller. The fact that amounts are sent over-the-wire as strings is an implementation detail that shouldn't leak out of this interface.